src/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+import routes from './reducer_routes';
+// all other reducers;
```

Comment:
Do we need this comment or is it implicit? If we keep it let's remove the semi colon at the end of the comment.